### DISCUSSION TITLE
Dark Mode Please

### DISCUSSION THREAD

**Discussion by joekincognito** [twbs/bootstrap#35889] [Ideas] [answered]
Proper OS-driven dark mode that changes bootstrap to dark mode based on OS settings please!! 
@media (prefers-color-scheme: dark) {

I, like many people probably, am working on switching everything to dark mode. I've switched my Windows to dark mode and it's nice. I noticed there are websites that auto-switched to dark mode after I changed it in my OS. Github does it, for instance. So I was looking into updating my bootstrap sites to respect the OS dark mode setting. It would be nice if bootstrap did it by default. Maybe v6?

Notes: Yes I know about Vino Rodrigues's bootstrap-dark and am appreciative of the work he's put in on that but I'm suggesting... just build it in.

**Comment by mdo** [comment, marked as answer]
https://github.com/twbs/bootstrap/pull/35857

**Reply by joekincognito** [reply]
Awesome, thanks Mark!
